_episodes/03-files.md

```diff
@@ -283,7 +285,7 @@ SRR307030_2.fastq
 >
 > * `gunzip` unzips .gz files
 > * `unzip` unzips .zip files
-> * `unrar` unzips .rar files
+> * `unrar` unzips .rar files (not available on many systems)
```

Comment:
if it is not available on many systems, maybe we should rethink this callout. Further, `unzips .rar files` could be rephrased to `extracts the contents of .rar files` perhaps.